### ISSUE TITLE
The "Previous", "Header view" and "Next" button are not accessible by the keyboard keys for calendar control.

### ISSUE THREAD

**Issue by msftedad** [ant-design/ant-design#43933]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antd-a11y-datepicker-nbx63c)

### Steps to reproduce

1. Launch Code: Antd a11y datepicker - CodeSandbox
2. Tab till Calendar control and press enter on it.
3. Observed that, the "Previous", "Header view" and "Next" button are not accessible by the keyboard keys for calendar control.

### What is expected?

The "Previous", "Header view" and "Next" button are not accessible by the keyboard keys for calendar control.

### What is actually happening?

The "Previous", "Header view" and "Next" button should be accessible by the keyboard keys for calendar control.

| Environment | Info |
| --- | --- |
| antd | undefined |
| React | 16.3 |
| System | Windows 11 Enterprise 22H2 Version: 22621.1992 |
| Browser | Edge Version 115.0.1901.188 (Official build) (64-bit) |

---
Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-rghuge_microsoft_com/EfqB4Evu98hJhEamQDqXi34BPF8J-BLBxQL-PGZxGlPWXg?e=vT2y1k&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZyIsInJlZmVycmFsQXBwUGxhdGZvcm0iOiJXZWIiLCJyZWZlcnJhbE1vZGUiOiJ2aWV3In19

**Comment by zombieJ** [comment]
It's by design. DatePicker provides hot key for access fast year, month change but not provide keyboard access to the popup panel since it's not easy for people to "hear" the UI content. So we just leave input here.

**Comment by msftedad** [comment]
Hi @zombieJ,  Calendar control is not accessible by keyboard keys. "Previous", "Header view", "Next" and "Dates" button are not accessible by keyboard keys user only access them by using mouse only. As the keyboard user not able to access the Calendar control using keyboard keys.

**Comment by msftedad** [comment]
Hi @zombieJ, Can we know the hot key for access fast year, month change provide by Date picker. Thanks

**Comment by msftedad** [comment]
Hi @zombieJ, Any update?

**Comment by yoyo837** [comment]
ref: https://github.com/react-component/picker/pull/691

**Comment by msftedad** [comment]
Hi @yoyo837 , Can we know the ETA to fix this issue?

**Comment by yoyo837** [comment]
> Hi @yoyo837 , Can we know the ETA to fix this issue?

https://github.com/ant-design/ant-design/pull/46685 cc @zombieJ

**Comment by msftedad** [comment]
Hi @yoyo837  , Can we know the ETA to fix this issue?

**Comment by Swati1700** [comment]
@yoyo837 , Can we know the ETA to fix this issue?

**Comment by yoyo837** [comment]
Sorry! The main members are all on the holidays and will resume work as soon as possible after the holidays.

**Comment by yoyo837** [comment]
https://github.com/ant-design/ant-design/pull/46685 has been released in 5.14.0, please try this version

**Comment by msftedad** [comment]
Thanks for the update.
